qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -34,6 +41,22 @@ public void setFileUrl(String fileUrl) {
 		WebElement element = driver.findElement(By.xpath("//input[@type='file']"));
 		element.sendKeys(fileUrl);
 	}
+	
+	private String createTemporaryFile() throws IOException {
+		File tempFile = File.createTempFile("file", ".pdf");
+		tempFile.deleteOnExit();
+		try (InputStream pdf = getClass().getClassLoader().getResourceAsStream("data/file.pdf")) {
+			try (Writer writer = new FileWriter(tempFile)) {
+				IOUtils.copy(pdf, writer, StandardCharsets.UTF_8);
+			}
+		}
+		return tempFile.getAbsolutePath();
+	}
+	
+	public void attachFile() throws IOException {
+		WebElement element = driver.findElement(By.xpath("//input[@type='file']"));
+		element.sendKeys(createTemporaryFile());
+	}
```

Comment:
So, picking up from where I left off:

```suggestion
	public void attachFile() throws IOException {
		setFileUrl(createTemporaryFile());
	}
```